src/main/java/com/onlinebookstore/model/Role.java

```diff
@@ -0,0 +1,33 @@
+package com.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    @Enumerated(EnumType.STRING)
+    private RoleName role;
+
+    @Override
+    public String getAuthority() {
+        return role.name();
+    }
+
+    public enum RoleName {
+        USER,
+        ADMIN
```

Comment:
Spring Security requires use a prefix ROLE_. You can add this prefix to your enum values or change method `getAuthority()` to return "ROLE_" + roleName.name(), but it's better just change enum values
```suggestion
        ROLE_USER,
        ROLE_ADMIN
```